packages/app-mobile/services/AlarmServiceDriver.android.ts

```diff
@@ -1,7 +1,7 @@
 import Logger from '@joplin/lib/Logger';
 import { Notification } from '@joplin/lib/models/Alarm';
 
-const ReactNativeAN = require('react-native-alarm-notification').default;
+const ReactNativeAN = require('joplin-rn-alarm-notification').default;
```

Comment:
If the package is generic enough, maybe you could even make it regular notification package? I'm sure many people would be interested.

Reply:
I'm not sure I want to support it for everyone. Plus I only know the android side and people will come with iOS issues.

Reply:
That makes sense, it would still be additional work to make it work in the general case.